content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,31 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and their predictions to make an educated decision about the identified predictive patterns, collectively called _ensemble learning_. 
```

Comment:
I feel like this is too long of an introduction & it might be better to explain very briefly why one might want an ensemble (a single predictor may not adequately capture the data) + discuss approaches that use models with the same parameters w/ different starting points (i.e., RF, k-means clustering) or those that use multiple distinct algorithms with different assumptions/etc.